server/services/courseService.test.ts

```diff
@@ -269,18 +316,24 @@ describe('CourseService', () => {
   describe('presentCourseParticipation', () => {
     const addedByUser = userFactory.build({ name: 'john smith' })
     const courseParticipation = courseParticipationFactory.build({ addedBy: addedByUser.username })
+    const addedByDisplayName = StringUtils.convertToTitleCase(addedByUser.name)
+    const referralId = 'a-referral-id'
 
-    it('calls the user service to get details of the user who added the course participation and returns the course participation with formatted `addedByDisplayName`', async () => {
+    it('calls the user service to get details of the user and converts it to summary list options', async () => {
```

Comment:
`converts the participation and user into summary list options`? In other utils we've usually described these methods as `formats [thing] in the appropriate format for passing to a GOV.UK summary list Nunjucks macro` so this could possibly be `calls the user service to get details of the user, then formats the participation and user in the appropriate format for passing to a GOV.UK summary list Nunjucks macro` if that's not too verbose

Reply:
Good point, have updated